gui/widget.cpp

```diff
@@ -119,10 +118,8 @@ void Widget::draw() {
 				_x = _x + g_gui.getOverlayOffset();
 			}
 
-			Common::Rect r = _boss->getClipRect();
-			r.moveTo(_x, r.top);
-
-			g_gui.theme()->swapClipRect(r);
+			clip.moveTo(_x, clip.top);
+			g_gui.theme()->swapClipRect(clip);
```

Comment:
I'm not entirely sure about this change, but it appears to work with Hebrew and English, and both the old and new style scrollcontainers in tabs

Reply:
This code is used in RTL languages. Since it works with Hebrew, it should be fine